### PR TITLE
 	Changed Donation repo to no longer return deleted donations

### PR DESCRIPTION
**Pull request by JeroenDeDauw**
Includes #322 

Note: I'm not actually sure this specific filter is correct. Though I am thinking we need to filter on several conditions, each which make the Donation no longer relevant for this app. Full list anyone?

### CODE REVIEW

**Comment by gbirke** [comment]
This needs a rebase, #322 was modified and merged.

**Comment by JeroenDeDauw** [comment]
It can still be merged no? I'll do a rebase anyway though am not sure what this gains us.

File: src/DataAccess/DoctrineDonationRepository.php [hunk 1]
```diff
@@ -207,15 +207,11 @@ public function getDonationById( int $id ) {
...
-			 * 'isPublic' => true,
-			 * 'dtDel' => null
-			 * correct status
-		 */
-
-		return $this->entityManager->find( DoctrineDonation::class, $id );
+	public function getDoctrineDonationById( int $id ) {
+		return $this->entityManager->getRepository( DoctrineDonation::class )->findOneBy( [
+			'id' => $id,
+			'dtDel' => null
```
Comment (JeroenDeDauw) [inline src/DataAccess/DoctrineDonationRepository.php]: This field is about deletion via the backend app right? Or am I confusing it with the cancellation stuff we have in this app?

Comment (KaiNissen) [inline src/DataAccess/DoctrineDonationRepository.php]: Ye Olde Application™ also sets this timestamp, when the dataset is updated with an inactive status (**D**eleted, **P**ending, in**X**omplete, promi**Z**ed bank transfer). I think only the first one really makes sense, the other three don't really relate to a deletion/cancellation.

I think both the deletion in the management/backend application and the cancellation by the user should trigger this timestamp to be set.

Comment (JeroenDeDauw) [inline src/DataAccess/DoctrineDonationRepository.php]: We talked about this and figured that is another task for another PR

**Pull request merged**